locuszoom.app.js

```diff
@@ -674,19 +737,38 @@ LocusZoom.Data.GeneSource.prototype.getURL = function(state, chain, fields) {
 LocusZoom.Data.GeneSource.prototype.parseResponse = function(resp, chain, fields, outnames) {
     return {header: chain.header, body: resp.data};
 };
-LocusZoom.Data.GeneSource.SOURCE_NAME = "GeneLZ";
+
+LocusZoom.Data.RecombinationRateSource = LocusZoom.Data.Source.extend(function(init) {
+    this.parseInit(init);
+}, "RecombLZ");
+
+LocusZoom.Data.RecombinationRateSource.prototype.getURL = function(state, chain, fields) {
+    var source = state.recombsource || chain.header.recombsource || this.params.source || 15;
+    return this.url + "?filter=id in " + source +
+        " and chromosome eq '" + state.chr + "'" + 
+        " and position le " + state.end +
+        " and position ge " + state.start;
+};
+
+LocusZoom.Data.StaticSource = LocusZoom.Data.Source.extend(function(data) {
```

Comment:
Thanks for setting this up. We can make this branch a dependency for #50 so that branch can use this pattern for defining static data instead of using the layout.